haskell/src/Ligo/BaseDAO/Types.hs

```diff
@@ -577,7 +577,7 @@ mkStorage admin extra metadata now tokenAddress qt =
         }
     , sTotalSupply = M.fromList [(frozenTokenId, 0)]
     , sFreezeHistory = mempty
-    , sStartTime = arg #now now
+    , sStartLevel = arg #now now
```

Comment:
Same here, with these changes `now` seems a bit out of place.

Reply:
Changed